configure.ac

```diff
@@ -1210,8 +1210,6 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <xmmintrin.h>]], [[
 
 AC_MSG_CHECKING(for strong getauxval support in the system headers)
```

Comment:
just remove 'strong' here imo

Reply:
Although we can't rename the define - it's used by the crc32c library after all. Maybe better to leave it as is I dunno,

Reply:
> Although we can't rename the define - it's used by the crc32c library after all.

Yes if that wasn't the case we could rename the define, and probably simplify the configure check further.

> Maybe better to leave it as is I dunno,

I think leaving `strong` in here is ok for now.